app/models/meter.rb

```diff
@@ -238,20 +236,20 @@ def can_withdraw_consent?
     consent_granted
   end
 
-  def self.csv_headers
-    ["School group", "School", "MPAN/MPRN", "Meter type", "Active", "First validated meter reading", "Last validated meter reading", "Admin Meter Status", "Open issues"]
+  def smart_meter_tariff_attributes
+    @smart_meter_tariff_attributes ||= Amr::AnalyticsTariffFactory.new(self).build
   end
 
-  def self.csv_attributes
-    %w{school.school_group.name school.name mpan_mprn meter_type.humanize active first_validated_reading last_validated_reading admin_meter_status_label open_issues_count}
+  def open_issues_count
+    open_issues.count
   end
 
-  def smart_meter_tariff_attributes
-    @smart_meter_tariff_attributes ||= Amr::AnalyticsTariffFactory.new(self).build
+  def open_issues_as_list
```

Comment:
I think it would best best to impose some default order on these, e.g. show the oldest issues as the first in the list.